### PR TITLE
[20] imported the data set

### PR DESCRIPTION
**Pull request by Yuji-github**
PR  practice

PR: Pull Request 
Purposes: double checking

### CODE REVIEW

File: src/eda.py
Comment (ayami-n) [inline src/eda.py:1]: can you make a space in your code convention? line 23 & 30

Comment (Yuji-github) [reply src/eda.py:1]: Done

Comment (ayami-n) [inline src/eda.py:1]: it should  be like this
:return: bool

Comment (Yuji-github) [reply src/eda.py:1]: Thank you for paying attention =)

**Review by ayami-n — APPROVED** [review]

**Pull request merged**